lib/generators/setup_generator.rb

```diff
@@ -176,13 +176,25 @@ def footer_logo
   end
 
   def gems
-    gem 'bootstrap-sass', '~> 3.3.6'
-    gem 'jquery-rails', '~> 4.3'
+    # Replace sass-rails with sassc-rails
+    gsub_file "#{@new_app}/Gemfile", /^(gem 'sass-rails'.*)$/,
+      "gem 'sassc-rails', '~> 2.1'"
 
     # Remove turbolinks gem
     gsub_file "#{@new_app}/Gemfile", /^(gem 'turbolinks'.*)$/, "#\\1"
 
-    return "Gems: Turbolinks removed"
+    # Replace chromedriver-helper with webdrivers
+    gsub_file "#{@new_app}/Gemfile",
```

Comment:
I think I was picturing that these steps would be taking place in some kind of generator like `update_x_to_x` or whathaveyou.  Most new Rails apps (like if we spun up a new Rails 5 app) wouldn't actually have chromedriver-helper and such things in them anymore, right?  I guess maybe older versions of Rails 5 would?  I'm not sure.

Reply:
I think 5.2.3 still uses these. That's why I wanted to add this stuff to our generator.

Reply:
okay